File: 6kyu/multiples_of_3_or_5.dart
void main() {
  int result = solution(200);
  print(result);
}

int solution(int n) {
  int answer = 0;
  List<int> multi35 = [];

  for (int i = 1; i < n; i++) {
    if (i % 3 == 0 || i % 5 == 0) {
      multi35.add(i);
    }
  }

  for (int num in multi35) {
    answer += num;
  }

  return answer;
}

File: 6ky/multiples-of-3-or-5.swift
func solution(_ num: Int) -> Int {
    var total:Int = 0
  
    guard num > 0 else { return 0 }
    
    for i in 1..<num {
        if (i % 3 == 0 || i % 5 == 0) {
            total += i
        }
    }
  
    return total
}
